arwen/host/execution_test.go

```diff
@@ -1140,53 +1060,6 @@ func TestExecution_AsyncCall_CallBackFails(t *testing.T) {
 	require.Equal(t, expectedVMOutput, vmOutput)
 }
 
-func TestExecution_AsyncCall_BuiltinFails(t *testing.T) {
```

Comment:
why did you delete these tests ?

Reply:
They were moved into `execution_builtin_test.go`.